libs/components/modals/src/lib/modules/confirm/confirm.service.spec.ts

```diff
@@ -10,25 +13,31 @@ describe('Confirm service', () => {
 
   beforeEach(() => {
     modalService = new MockSkyModalService();
-    confirmService = new SkyConfirmService(modalService as any);
+    confirmService = new SkyConfirmService(
```

Comment:
This might be a good opportunity to refactor this code to use `TestBed` to construct `SkyConfirmService`. Then we can start using `inject()` instead of constructor parameters in that service.